src/mame/drivers/vt1682.cpp

```diff
@@ -3999,6 +4259,59 @@ WRITE8_MEMBER(vt_vt1682_state::vt1682_2137_alu_div_opr6_trigger_w)
     0x01 - TMRA EN
 */
 
+READ8_MEMBER(vt_vt1682_state::vt1682_soundcpu_2102_timer_a_enable_r)
+{
+	uint8_t ret = m_soundcpu_2102_timer_a_enable;
+	logerror("%s: vt1682_soundcpu_2102_timer_a_enable_r returning: %02x\n", machine().describe_context(), ret);
+	return ret;
+}
+
+WRITE8_MEMBER(vt_vt1682_state::vt1682_soundcpu_2102_timer_a_enable_w)
+{
+	// For NTSC
+
+	//Period = (65536 - Timer _PreLoad) / 21.4772 MHz
+	//Timer PreLoad = 65536 � (Period in seconds) * 21.4772 * 1000000 )
+
+	// For PAL
+	// Period = (65536 - Timer PreLoad) / 26.601712 MHz
+	//Timer PreLoad = 65536 � (Period in seconds) * 26.601712 * 1000000 )
```

Comment:
Please ensure your source files are valid UTF-8.  It's not fun having to guess the encoding of a source file and clean this stuff up at srcclean time.

Reply:
ah sorry, I did run this through srcclean before submitting, guess word mangled my notes or something

Reply:
anyway I've fixed it locally, so I'll be correct in the next PR